frontend/device/remarkable/device.lua

```diff
@@ -112,12 +113,30 @@ end
 
 
 function Remarkable:init()
+    -- Check if we were launched from Oxide launcher
+    --- @fixme this could be changed to check parent process of KOReader instead
+    local oxide_running = os.execute("systemctl is-active --quiet tarnish") == 0
+    logger.info(string.format("Oxide running?: %s", oxide_running))
+
+    -- experiment
+    -- logger.info("PPID:")
+    -- local parent_process = os.execute("echo $PPID")
+    -- os.execute("ps | grep $PPID")
+    -- logger.info(string.format("parent proccess is oxide?: %s", parent_process_is_oxide))
+
     self.screen = require("ffi/framebuffer_mxcfb"):new{device = self, debug = logger.dbg}
     self.powerd = require("device/remarkable/powerd"):new{
         device = self,
         capacity_file = self.battery_path,
         status_file = self.status_path,
     }
+
+    local event_map = require("device/remarkable/event_map")
+    -- If we were launched from Oxide, remove Power from the event map
```

Comment:
Something like:
```suggestion
    -- If we are launched while Oxide is running, remove Power from the event map
```

Reply:
Sure, but the "were" was like 2 milliseconds ago.

Reply:
That makes sense